src/storage/config.rs

```diff
@@ -38,13 +39,13 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Config {
-        let total_cpu = sysinfo::get_logical_cores();
+        let cpu_num = SysQuota::new().cpu_cores_quota();
         Config {
             data_dir: DEFAULT_DATA_DIR.to_owned(),
             gc_ratio_threshold: DEFAULT_GC_RATIO_THRESHOLD,
             max_key_size: DEFAULT_MAX_KEY_SIZE,
             scheduler_concurrency: DEFAULT_SCHED_CONCURRENCY,
-            scheduler_worker_pool_size: if total_cpu >= 16 { 8 } else { 4 },
+            scheduler_worker_pool_size: if cpu_num >= 16 { 8 } else { 4 },
```

Comment:
I think `16` is too large... Because every key which is checked by `scheduler` would insert exactly once in apply-thread. But apply-thread has to deal with data of three replicas.

Reply:
I suggest changing it to `if cpu_num >= 16 { 6 } else { 3 }`

Reply:
This is not related to this PR, maybe another PR.